### PR TITLE
Add note about code style for ui_next

### PR DESCRIPTION
**Pull request by nixocio**
Add note about code style for `ui_next`.

### CODE REVIEW

File: awx/ui_next/CONTRIBUTING.md [hunk 1]
```diff
@@ -6,31 +6,43 @@ Have questions about this document or anything not covered here? Feel free to re
...
+      - [Naming components that use the context api](#naming-components-that-use-the-context-api)
+    - [Class constructors vs Class properties](#class-constructors-vs-class-properties)
+    - [Binding](#binding)
+    - [Typechecking with PropTypes](#typechecking-with-proptypes)
+    - [Naming Functions](#naming-functions)
+    - [Default State Initialization](#default-state-initialization)
+    - [Testing components that use contexts](#testing-components-that-use-contexts)
+  - [Internationalization](#internationalization)
+    - [Marking strings for translation and replacement in the UI](#marking-strings-for-translation-and-replacement-in-the-ui)
+    - [Setting up .po files to give to translation team](#setting-up-po-files-to-give-to-translation-team)
```
Comment (nixocio) [inline awx/ui_next/CONTRIBUTING.md]: For whatever reason my editor is modifying this block. Let me update this.

File: awx/ui_next/CONTRIBUTING.md [hunk 2]
```diff
@@ -31,6 +31,10 @@ Have questions about this document or anything not covered here? Feel free to re
 - All code submissions are done through pull requests against the `devel` branch.
 - If collaborating with someone else on the same branch, please use `--force-with-lease` instead of `--force` when pushing up code. This will prevent you from accidentally overwriting commits pushed by someone else. For more information, see https://git-scm.com/docs/git-push#git-push---force-with-leaseltrefnamegt
 - We use a [code formatter](https://prettier.io/). Before adding a new commit or opening a PR, please apply the formatter using `npm run prettier`
+- We adopt the following code sytle guide:
```
Comment (marshmalien) [inline awx/ui_next/CONTRIBUTING.md]: Small spelling error `sytle`.

**Review by marshmalien — CHANGES_REQUESTED** [review]
Thanks for adding this to the docs!

**Review by marshmalien — APPROVED** [review]

**Comment by jakemcdermott** [comment]
regate

**Comment by marshmalien** [comment]
regate

**Pull request merged**